Patches/PsiTech/ThingDefs/Equipment/PsiTechBasicApparel.xml

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>PsiTech</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<!-- == PTRobeBase == -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[@Name="PTRobeBase"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<StuffEffectMultiplierArmor>2</StuffEffectMultiplierArmor>
```

Comment:
I assume this is a cloth/leather robe/hood? 2mm is basically t-shirt material. This could probably be increased to 4, if you think it's appropriate.

Reply:
Yes, this is made from fabrics (cloth, devilstrand and similar). I guess I will up it to 4 then.